test-integration/samples/.jhipster/MapsIdGrandchildEntityWithoutDTO.json

```diff
@@ -0,0 +1,24 @@
+{
+  "changelogDate": "20220313234416",
+  "clientRootFolder": "",
+  "databaseType": "sql",
+  "dto": "no",
+  "entityTableName": "maps_id_grandchild_wo_dto",
+  "fields": [{ "fieldName": "date", "fieldType": "Instant" }],
+  "fluentMethods": true,
+  "jpaMetamodelFiltering": false,
+  "pagination": "no",
+  "relationships": [
+    {
+      "id": true,
+      "otherEntityField": "id",
+      "otherEntityName": "mapsIdChildEntityWithDTO",
+      "otherEntityRelationshipName": "mapsIdGrandchildEntityWithDTO",
```

Comment:
```suggestion
```

Reply:
`"otherEntityRelationshipName"` is needed even if the relationship is unidirectional, right?